h/views/account_signup.py

```diff
@@ -1,7 +1,7 @@
 import datetime
 from typing import Any
 
-import deform
+from deform import ValidationFailure
```

Comment:
Clarify that `deform` is not really used in this file, other than catching `ValidationFailure`.